eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -711,7 +714,7 @@ function UpdatePackageVersions($pkgWorkItem, $plannedVersions, $shippedVersions)
     }
   }
 
-  foreach ($version in $plannedVersionSet.Keys)
+  foreach ($version in @($plannedVersionSet.Keys))
```

Comment:
Won't this have the unintended (I assume) effect of adding a single iteration when `Keys` is empty/null?

Reply:
This just copies the keys and if they are empty it will be an empty array. It won't add an interaction for null.